docs/self-managed/operational-guides/update-guide/870-to-880.md

```diff
@@ -6,6 +6,140 @@ description: "Review which adjustments must be made to migrate from Camunda 8.7.
 
 The following sections explain which adjustments must be made to migrate from Camunda 8.7.x to 8.8.x for each component.
 
+## Camunda Exporter
+
+The new Camunda Exporter brings the importing and archiving logic of web components (Tasklist and Operate) closer to the distributed platform (Zeebe). This simplifies the installation, enables scalability for the web applications, reduces latency when showing runtime and historical data, and reduces data duplication (resource consumption).
+
+![87-state](./img/87-state-refined.png)
+
+Camunda web applications use importers and archivers to consume, aggregate, and archive historical data provided by the Elasticsearch (ES) or OpenSearch (OS) exporters. The new Camunda Exporter introduces changes in the architecture users should be aware of.
+
+During the migration/update phase, the Operate and Tasklist importer and the Elasticsearch exporter are still required. This allows 8.7 data written to the Zeebe journal, or already in the Zeebe Elasticsearch indices, to be imported into the new harmonized index schema.
+
+![brownfield](./img/target-brown-field-refined.png)
+
+### Harmonized Indices
+
+The harmonized schema looks like the following:
+
+![harmonized-schema](./img/harmonized-indices-miro-v2.png)
+
+This reduces the need for several indices and to make use of existing ones (Operate indices are used by Tasklist, etc.)
```

Comment:
Since the content is largely the same, I think it would be appropriate to just link directly to the announcement here, and that way this guide can only include the upgrade notes. This would allow you to increase the headers on the later content and make it a bit easier to browse.